config/mcp_config.py

```
@@ -0,0 +1,116 @@
+        for name, url in endpoints.items():
+            if url:
+                url_lower = url.lower()
+                for indicator in mock_indicators:
+                    if indicator in url_lower and "no-mock" not in url_lower:
+                        raise ValueError(
+                            f"Mock indicator '{indicator}' found in {name}: {url}"
+                        )
+
+        return True
+
+    def validate_config(self) -> Dict[str, bool]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)